### DISCUSSION TITLE
[A-L] (2023/24) Foglio 2 - Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#143] [Esercizi - Bertini] [answered]
![Es 4](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/f2579e88-2eda-4d0e-9984-d1405af5c146)

**Comment by daromacarlo** [comment, marked as answer]
![Immagine 2023-10-14 181650](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/cfa0bc66-2994-4267-9d34-20d3a7287607)

**Comment by CiottoloMaggico** [comment]
Analogo @CarloDaRomadev

**Comment by niccolo-fato** [comment]
Analogo a @CarloDaRomadev
